new.css

```diff
@@ -0,0 +1,316 @@
+body {
```

Comment:
re-implement this with SASS CSS pre-compiler.

Reply:
Learning Basics about SASS.